doc/from-zero-to-hacking.md

```diff
@@ -1,66 +1,109 @@
 # From Zero to Hacking
 
-## Note Before
+The goal of this document is to guide you through the process of setting
+up a local, VM-based test instance via `Vagrant` on a Linux system of
+your choice.
+
+This includes checking out all the required repositories, setting up
+a local build environment, running your first image build, and
+starting the resulting cluster.
+
+After only about half an hour, you will be able to login to a Ceph
+cluster managed via `Aquarium` for the first time!
+
+## Requirements
+
+The requirements on your environment to follow this guide are as
+follows:
+
+- A physical machine, *not* a VM
+  - Currently, this guide is tested on `x86_64`, but `aarch64` is
+    coming!
+- At least 16 GB of disk space
+  - Approximately 4 GB for the directory you're building Aquarium in,
+    and another 4 GB for `/var/lib/libvirt/` (or equivalent on your
+    system). Note that this will grow if you actually store data in the
+    cluster.
+- Preferably [openSUSE Leap](https://www.opensuse.org/#Leap) or
+  [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed) as the
+  host, though we're working on extending this guide and our tooling to
+  support other distributions - patches more than welcome!
```

Comment:
Might be worth adding note that the next system we're planning on testing on is Ubuntu, and we're open to patches?

Reply:
I didn't want to mix the document with the roadmap. But I have opened an issue for tracking: https://github.com/aquarist-labs/aquarium/issues/223